IRremoteESP8266.cpp

```diff
@@ -702,23 +702,32 @@ static void ICACHE_RAM_ATTR read_timeout(void *arg __attribute__((unused))) {
 
 static void ICACHE_RAM_ATTR gpio_intr() {
   uint32_t gpio_status = GPIO_REG_READ(GPIO_STATUS_ADDRESS);
+  uint32_t now = system_get_time();
```

Comment:
Why system_get_time() instead if micros()? Curious. Suggesting the later as that is what is used elsewhere in the code.